app/models/operating_system.rb

```diff
@@ -28,7 +28,9 @@ class OperatingSystem < ApplicationRecord
     ["linux_coreos",    %w(coreos)],
     ["linux_esx",       %w(vmnixx86 vmwareesxserver esxserver vmwareesxi)],
     ["linux_solaris",   %w(solaris)],
-    ["linux_generic",   %w(linux)]
+    ["linux_generic",   %w(linux)],
+    ["ibm_aix",         %w(aix)],
```

Comment:
SInce the rest of these are `linux_#{flavor}` wouldn't `unix_aix` fit better?

Reply:
Sure, we can rename AIX. In this case I'll do a follow-up commit to the MIQ **decorators** repository as the corresp. filename then has to be changed.

Reply:
here the pull request for the decorators: https://github.com/ManageIQ/manageiq-decorators/pull/30